### PR TITLE
[dbus] Upgrade to 1.13.2 , fix libdir

### PR DESCRIPTION
**Pull request by lovedheart**
#1981

### CODE REVIEW

File: packages/dbus.rb [hunk 1]
```diff
@@ -3,27 +3,27 @@
...
   })
 
   depends_on 'expat'
 
   def self.build
-    system "./configure"
+    system "./configure \
+    --prefix=#{CREW_PREFIX} \
+    --libdir=#{CREW_LIB_PREFIX} \
+    --localstatedir=#{CREW_DEST_PREFIX}/var \
```
Comment (uberhacker) [inline packages/dbus.rb]: Not sure if you are aware but this `--localstatedir` directory is read-only on CloudReady.  You might notice an error during installation.  This is usually a good indicator of the problem.

**Comment by uberhacker** [comment]
Thank you for the update @lovedheart.  Please see inline comment.

File: packages/dbus.rb (hunk 1 above)
Comment (lovedheart) [reply packages/dbus.rb]: @uberhacker Remove --localstatedir ?

Comment (uberhacker) [reply packages/dbus.rb]: You could remove or just change to a writable directory like `#{CREW_DEST_PREFIX}/tmp` instead.

**Pull request merged**